File: DsaWithCpp/Tree/AVLTree/Node.cpp
struct Node
{
    int data, height;
    Node *left;
    Node *right;

    Node()
    {
    }

    Node(int val)
    {
        data = val;
        height = height;
        left = right = NULL;
    }

    Node(int val, Node *leftptr, Node *rightptr)
    {
        data = val;
        height = height;
        left = leftptr;
        right = rightptr;
    }
};

File: Java/tree/binarysearchtree/Node.java
package tree.binarysearchtree;

public class Node {
    int data;
    Node left;
    Node right;

    Node() {

    }

    Node(int data) {
        this.data = data;
        this.left = this.right = null;
    }

    Node(int data, Node left, Node right) {
        this.data = data;
        this.left = left;
        this.right = right;
    }
}
